taxi/tests/test_models.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from taxi.models import Manufacturer, Car
+
+
+class ModelsTest(TestCase):
+    def test_manufacturer_str(self):
+        manufacturer = Manufacturer.objects.create(
+            name="test",
+            country="test country"
+        )
+        self.assertEqual(
+            str(manufacturer),
+            f"{manufacturer.name} {manufacturer.country}"
+        )
+
+    def test_create_driver_license_number(self):
+        username = "test"
+        password = "1234"
+        license_number = "TTT12345"
+        driver = get_user_model().objects.create_user(
+            username=username,
+            password=password,
+            license_number=license_number
+        )
+        self.assertEqual(driver.username, username)
+        self.assertTrue(driver.check_password(password))
+        self.assertEqual(driver.license_number, license_number)
+
+    def test_driver_str(self):
+        driver = get_user_model().objects.create(
```

Comment:
Consider using `get_user_model().objects.create_user` instead of `create` to ensure that the password is hashed correctly when creating a driver.